docker/actinia-core-tests/actinia-test-worker-usedby-worker.cfg

```diff
@@ -0,0 +1,31 @@
+[GRASS]
+grass_database = /tmp/actinia_core/grassdb
+grass_user_database = /tmp/actinia_core/userdata
+grass_tmp_database = /tmp/actinia_core/workspace/temp_db
+grass_resource_dir = /tmp/actinia_core/resources
+
+[API]
+force_https_urls = True
+
+[LIMITS]
+max_cell_limit = 22500000
+process_time_limt = 60
+process_num_limit = 20
+
+[REDIS]
+worker_logfile = /tmp/actinia_core/workspace/actinia_worker_test.log
+redis_server_url = localhost
+redis_server_port = 6379
+
+[QUEUE]
+queue_type = redis
+number_of_workers = 1
+redis_queue_server_url = localhost
+worker_queue_prefix = job_queue
+
+[LOGGING]
+log_level = 1
+
+[MISC]
+tmp_workdir = /tmp/actinia_core/workspace
+download_cache = /tmp/actinia_core/workspace/download_cache
```

Comment:
Sind die beiden Configs nicht gleich?

Reply:
Yes they are equal - as preparation to have tests where they could differ.